package.json

```diff
@@ -79,19 +80,20 @@
     "react-router-config": "^4.4.0-beta.6",
     "react-router-dom": "^4.4.0-beta.6",
     "react-select": "^1.3",
-    "reactstrap": "^6.4.0",
+    "reactstrap": "^8.5.1",
```

Comment:
We need to bump the dependency here to get the toast component from Reacstrap

Reply:
This is kind of dangerous. We've had problems before with Reactstrap changes in core. In theory modules should bring their own Reactstrap versions of course. Let's just make sure we give it enough time after merge to test thoroughly before releasing.